src/sections/featured-product.liquid

```diff
@@ -59,79 +111,65 @@
               id="SingleOptionSelector-{{ section.id }}-{{ forloop.index0 }}"
               data-single-option-selector
               data-index="option{{ option.position }}">
-              {% for value in option.values %}
+              {%- for value in option.values -%}
                 <option
                   value="{{ value | escape }}"
                   {% if option.selected_value == value %}selected="selected"{% endif %}>
                     {{ value }}
                 </option>
-              {% endfor %}
+              {%- endfor -%}
             </select>
           </div>
-        {% endfor %}
-      {% endunless %}
+        {%- endfor -%}
+      {%- endunless -%}
 
-      <select name="id" class="no-js" data-product-select>
-        {% for variant in product.variants %}
+        <select name="id" class="no-js" data-product-select>
```

Comment:
I didn't check the demo store yet, but is the `<label for>` missing? I couldn't find it...